go/enclave/enclave.go

```diff
@@ -22,24 +22,25 @@ import (
 
 	"github.com/obscuronet/go-obscuro/go/common/log"
 
-	obscurocrypto "github.com/obscuronet/go-obscuro/go/enclave/crypto"
-
 	"github.com/obscuronet/go-obscuro/go/enclave/bridge"
 	"github.com/obscuronet/go-obscuro/go/enclave/rollupchain"
 
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/crypto/ecies"
 	"github.com/obscuronet/go-obscuro/go/config"
 
-	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/obscuronet/go-obscuro/go/common"
-	obscurocore "github.com/obscuronet/go-obscuro/go/enclave/core"
+
 	"github.com/obscuronet/go-obscuro/go/enclave/db"
 	"github.com/obscuronet/go-obscuro/go/enclave/mempool"
 	"github.com/obscuronet/go-obscuro/go/ethadapter/erc20contractlib"
 	"github.com/obscuronet/go-obscuro/go/ethadapter/mgmtcontractlib"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	obscurocore "github.com/obscuronet/go-obscuro/go/enclave/core"
```

Comment:
Shouldn't we use `gethcore` and `gethcrypto` instead of aliasing these, for consistency?

Reply:
I didn't create aliasing, but makes sense, I can fix it 👍